executor/src/main/java/com/redhat/service/bridge/executor/ExecutorsService.java

```diff
@@ -42,19 +41,14 @@ public void init(@Observes StartupEvent ev) {
             CloudEvent cloudEvent = CloudEventUtils.decode(message.getPayload());
             BridgeCloudEventExtension bridgeCloudEventExtension = ExtensionProvider.getInstance().parseExtension(BridgeCloudEventExtension.class, cloudEvent);
             String bridgeId = bridgeCloudEventExtension.getBridgeId();
-            Set<Executor> executors = executorsProvider.getExecutors(bridgeId);
-            if (executors == null) {
-                LOG.info("[executor] A message for BridgeID '{}' has been received, but no executors were found.", bridgeId);
-            }
-            if (executors != null) {
-                for (Executor e : executors) {
-                    try {
-                        e.onEvent(cloudEvent);
-                    } catch (Throwable t) {
-                        // Inner Throwable catch is to provide more specific context around which Executor failed to handle the Event, rather than a generic failure
-                        LOG.error("[executor] Processor with id '{}' on bridge '{}' failed to handle Event. The message is acked anyway.", e.getProcessor().getId(),
-                                e.getProcessor().getBridge().getId(), t);
-                    }
+            Executor executor = executorsProvider.getExecutor();
+            if (executor.getProcessor().getBridge().getId().equals(bridgeId)) {
```

Comment:
At the moment, all the events are pushed to the `events` topic. So every executor will receive the messages for every BridgeId. This is going to change as soon as we provision a new kafka topic for every bridge instance